samcli/commands/local/lib/swagger/parser.py

```diff
@@ -62,15 +62,13 @@ def get_apis(self):
 
         Returns
         -------
-        list of samcli.commands.local.lib.provider.Api
+        list of list of samcli.commands.local.apigw.local_apigw_service.Route
             List of APIs that are configured in the Swagger document
         """
 
         result = []
         paths_dict = self.swagger.get("paths", {})
 
-        binary_media_types = self.get_binary_media_types()
```

Comment:
Are we not reading binary media types out of the swagger?

Reply:
We are still reading the binary_media_types inside the swagger, but in a different way. Before, each route had the binary_media_types information instead of keeping a global copy. When we are extracting the swagger route in cfn_base_api_provider, we add it to the Api object's global copy.